KalturaDemos/BasicPlayerDemo/src/main/java/com/kaltura/basicplayerdemo/MainActivity.java

```diff
@@ -354,8 +354,15 @@ public void onKPlayerError(PlayerViewController playerViewController, KPError er
 
     @Override
     public void onKPlayerPlayheadUpdate(PlayerViewController playerViewController, long currentTime) {
-        mSeekBar.setProgress((int) (currentTime / playerViewController.getDurationSec() * 100));
-        Log.d(TAG, "onKPlayerPlayheadUpdate currentTime " + currentTime);
+        long currentSeconds = (int) (currentTime / 1000);
+        long totalSeconds = (int) (playerViewController.getDurationSec());
+
+        Double percentage = (double) 0;
+        if (totalSeconds > 0) {
+            percentage = (((double) currentSeconds) / totalSeconds) * 100;
+        }
+        Log.d(TAG, "onKPlayerPlayheadUpdate " +  currentSeconds + "/" + totalSeconds + " => " + percentage.intValue() + "%");
+        mSeekBar.setProgress(percentage.intValue());
```

Comment:
After changing the above to `double`, change this from `percentage.intValue()` to `(int)percentage`.